js/BoxMenuGroupView.js

```diff
@@ -11,6 +11,16 @@ class BoxMenuGroupView extends MenuItemView {
     _.defer(this.addChildren.bind(this));
     this.$el.imageready(this.setReadyStatus.bind(this));
     this.$el.parents('.boxmenu__item-container').addClass('has-groups');
+    this.updateItemCount();
+  }
+
+  updateItemCount() {
+    const models = this.model.getChildren().models;
+    models.forEach(model => {
+      model.set({
+        _totalChild: models.length
```

Comment:
Could set `models.length` to a variable to avoid recalculating it with every loop?
```suggestion
    const models = this.model.getChildren().models;
    const totalChildren = models.length;
    models.forEach(model => {
      model.set({
        _totalChild: totalChildren
```